.github/workflows/test.yml

```diff
@@ -25,7 +25,16 @@ jobs:
     container: obolibrary/odklite
     steps:
       - uses: actions/checkout@v4
+
       - run: check-rdfxml gfo-light.owl
       - run: check-rdfxml modules/gfo-base.owl
+
       - run: robot reason --input gfo-light.owl
       - run: robot reason --input modules/gfo-base.owl
+
+      # - run: |
+      #     result=$(robot report --input gfo-light.owl --profile report_profile.txt)
+      #     echo "::notice file=gfo-light.owl::$result"
```

Comment:
Should be removed before merging because its dead code and might make future maintenance harder.

Reply:
This is a placeholder for the upcoming gfo-light changes (#34).